### PR TITLE
[DE] Translation Map-Locations

### PR DESCRIPTION
**Pull request by DerSchlupf**
Final Version:
image
image
image
image

### CODE REVIEW

**Comment by DerSchlupf** [comment]
commit po4a

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
Okay i dont know, why arent the checks continuing?

File: translations/all_map_locations.tex/de.po [hunk 1]
```diff
@@ -440,6 +656,23 @@ msgid ""
...
+"    \\caption{\\small Kategorie: \\textbf{Mehrfach besuchbar}\\\\\n"
+"      \\textbf{Choose one}: \\protect\\pagelink{Trading}{Handle} Ressourcen ODER Entferne genau eine Karte ODER kaufe eine \\protect\\pagelink{War Machines}{Kriegsmaschine}.\n"
+"    }\n"
+"  \\end{minipage}\\hfill\n"
+"  \\begin{minipage}[t]{0.47\\textwidth}\n"
+"    \\vspace{0pt}\n"
+"    \\centering\n"
+"    \\phantom{j}\\pagetarget{War Machine Factory}{\\textbf{Kriegsmaschinenfabrik}}\\par\n"
+"    \\framedimage[\\textwidth]{\\map_locations/war_machine_factory.jpg}\n"
+"    \\caption{\\small Kategorie: \\textbf{Mehrfach besuchbar}\\\\Kaufe eine \\protect\\pagelink{War Machines}Kriegsmaschine (zum günstigeren Preis)}.\\phantom{.......}}\n"
```
Comment (qwrtln) [inline translations/all_map_locations.tex/de.po:674]:
```suggestion
"    \\caption{\\small Kategorie: \\textbf{Mehrfach besuchbar}\\\\Kaufe eine \\protect\\pagelink{War Machines}Kriegsmaschine (zum günstigeren Preis)}.\n"
```
There's an extra `}`, which prevents the document from compiling.

**Comment by qwrtln** [comment]
> Okay i dont know, why arent the checks continuing?

Bot can't trigger them.

**Comment by DerSchlupf** [comment]
> > Okay i dont know, why arent the checks continuing?
> 
> Bot can't trigger them.

Thanx for repairing the error :) Okay some lines too much and because of this many pages more :D I will fix it in the next days.

**Comment by DerSchlupf** [comment]
Question: Would it be possible to move the headline "Orte auf der Abenteuerkarte" in the center of the box? I think about other headlines, for example "Übersicht aller Orte" or something to short it, but It would be good to know if I can access two lines.

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by qwrtln** [comment]
Same as always: bot can't trigger them. Click "Update branch" and then `git pull`. Afterwards, you should be able to use the two-line header.

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by qwrtln** [comment]
You're playing ping-pong with the bot, where you remove it, and the bot adds it back.

**Comment by DerSchlupf** [comment]
yeah i try different small things but nothing works. This is how it looks like. Would you say its correct now?
image

**Comment by qwrtln** [comment]
Looks good

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by qwrtln** [comment]
To trigger the workflows, do this: `git commit --allow-empty -m "Trigger workflows" && git push`

**Comment by DerSchlupf** [comment]
Okay, i have no idea or too less skill to find the error. It would be great, if someone has the time in the next days to search it. For today its enough for me.

**Comment by DerSchlupf** [comment]
> To trigger the workflows, do this: `git commit --allow-empty -m "Trigger workflows" && git push`

I've done it

**Comment by DerSchlupf** [comment]
Okay there is anywhere a typo or another error. If someone could find it for me, it would be great. I have no idea where to search. But no haste: For today i will leave. Bye!

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
Okay one small fix is necessary. Printable page is broken. I do it later and post the pages later, too.

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
In German we would said: That was a very difficult birth. 
I'm ready with this section and it is ready for merge, in my opinion. Final version in the first post and printable rulebook is also checked and looks fine!

**Review by qwrtln — APPROVED** [review]
Well done

**Pull request merged**